### DISCUSSION TITLE
Versions > 0.61.12 remove note icon from internal links

### DISCUSSION THREAD

**Discussion by rodrigoalexandrecosta** [TriliumNext/Trilium#4454] [Q&A] [answered]
Like the title says, I did some tests with 0.61.13 and the betas from 0.62 and they all remove the note icon from a note's internal link, like this:
![image](https://github.com/zadam/trilium/assets/23075659/642c62ab-8ece-426b-aa1a-db027ccc65a6)

Is there any label, relation or other config that can return this to the original way:
![image](https://github.com/zadam/trilium/assets/23075659/323d1c86-11e9-4c48-bbb1-72581343185e)

**Comment by zadam** [comment, marked as answer]
Hello, this is a bug in the 0.61.13 version, will be fixed soon.

**Reply by rodrigoalexandrecosta** [reply]
Thank you very much!
